server/main_server/src/agent_main.cpp

```diff
@@ -145,15 +148,19 @@ auto main(int _argc, char* _argv[]) -> int
 
     // clang-format off
     opts_desc.add_options()
-        ("hostname-cache-shm-name,x", po::value<std::string>(), "")
-        ("dns-cache-shm-name,y", po::value<std::string>(), "")
```

Comment:
what were these x and y?

Reply:
They are the short option names for the long option names to the left of the comma. They aren't used.